refextract/app.py

```diff
@@ -107,12 +107,41 @@ def extract_references_from_file_url(args):
             )
         return jsonify({"extracted_references": extracted_references})
 
+    @app.route("/extract_references_from_list", methods=["POST"])
+    @parser.use_args(
+        {
+            "raw_references": fields.List(fields.String, required=True),
+            "journal_kb_data": fields.Dict(required=True),
+        },
+        locations=("json",),
+    )
+    def extract_references_from_list(args):
+        references = args.pop("raw_references")
+        journal_kb_data = args.pop("journal_kb_data")
+        journal_dict = {"journals": journal_kb_data}
+        extracted_references = []
+        try:
+            for reference in references:
+                extracted_reference = extract_references_from_string(
+                    reference,
+                    override_kbs_files=journal_dict,
+                    reference_format="{title},{volume},{page}",
+                )
+                if extracted_reference:
```

Comment:
I think the size we return should be the same as the raw to be merged correctly in next

Reply:
I dont see any restriction related to the size https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/workflows/tasks/actions.py#L502-L504

which will call https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/refextract/matcher.py#L112 

or am I missing something ?

Reply:
ok I see! I will return the raw then also in that case